app/experimenter/nimbus-ui/src/components/PageHome/Directory.tsx

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect, useState } from "react";
+import { useQuery } from "@apollo/client";
+import { GET_ALL_EXPERIMENTS } from "../../gql/experiments";
+import { GetAllExperiments_experiments } from "../../types/GetAllExperiments";
+import { humanDate, addDaysToDate } from "../../lib/dateUtils";
+
+const Directory: React.FunctionComponent = () => {
+  const { data } = useQuery<{
+    experiments: GetAllExperiments_experiments[];
+  }>(GET_ALL_EXPERIMENTS);
```

Comment:
Until we implement "real" pagination, we might want to [slice this](https://graphql.org/learn/pagination/#slicing) at 5-10 experiments? Or maybe not, I'm just thinking of 20 or 30 test experiments loading in staging potentially slowing things down.

Reply:
I'd be in favour of just listing them all until it becomes a problem (at which point we could land a change to slice it)